changelog/update-sandbox-mode-notice-tooltip-copy

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: update
+Comment: Just updating the copy of the tooltip.
+
+
```

Comment:
There is one extra line here that can be removed.

Also, does this message show up in the list of changes when pushing a new version? If yes, then we should make the message a little user-friendly.

Reply:
That line is generated by the command, so I would rather leave it there. Who knows how it might mess up the parser?! 🤷 

About the comment, that is not going into the changelog; that is a development comment you need to provide when  you don't want to add an entry in the release changelog. The entry that does is the one without any `Thing:` in front (in my case, an empty line).